targets/python/templates/Errors.py.ejs

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+# Error codes returned by PlayFabAPIs
+class PlayFabErrorCode(Enum):
+    Success = 0,
+    Unknown = 1,
+    ConnectionError = 2,
+    JsonParseError = 3,
+    <% for(var i=1; i<errorList.length-1; i++) { var errorProps = errors[errorList[i]] %><%- errorProps.name %> = <%- errorProps.id %>,
+    <% } %><% var errorProps = errors[errorList[errorList.length-1]] %><%- errorProps.name %> = <%- errorProps.id %>
+
+class PlayFabError:
+
+    def __init__(self, *args):
+        if len(args) == 1:
+            self.fromJson(args[0])
+        else:
+            self.HttpCode = 0
+            self.HttpStatus = ""
+            self.Error = ""
+            self.ErrorCode = PlayFabErrorCode.Unknown
+            self.ErrorMessage = ""
+            self.ErrorDetails = None# dictionary of string keys and list of strings for values
+
+    def fromJson(self, other):
+        self.HttpCode = other["code"]
+        self.HttpStatus = other["status"]
+        self.Error = other["error"]
+        self.ErrorCode = other["errorCode"]
+        self.ErrorMessage = other["errorMessage"]
+        self.ErrorDetails = other.get("errorDetails", None)
+
+    def GenerateErrorReport(self):
+        str = ""
+        if self.ErrorMessage != None:
+            str = self.ErrorMessage
+        if self.ErrorDetails == None:
+            return str
+
+        for pair in self.ErrorDetails:
+            for eachMsg in pair.Value: # we want to loop through each item in the dictionary here
+                str += "{}: {}".format(pair.Key, eachMsg)
+
+        return str
+
+    def __str__(self):
+        return self.GenerateErrorReport()
+
+class PlayFabException(Exception):
```

Comment:
This should literally just be:
class PlayFabException(Exception):
    pass
Everything after this line should be removed.
Make sure any raised exceptions are of this type